meerk40t/gui/settings.py

```diff
@@ -136,6 +136,14 @@ def __init__(self, *args, **kwds):
                 "classify_reverse",
                 False,
             ),
+            (
+                _("Legacy Classify"),
+                _(
+                    "Use the legacy classification algorithm rather than the modern classification algorithm."
```

Comment:
```
_(
    "Use the legacy classification algorithm rather than the modern classification algorithm."
) + "\n\n"
+ _(
    "Legacy classification is compatible with Meerk40t 0.6.22+."
) + "\n"
+ _(
    "Modern classification has better handling of raster paths."
)
```